tokio/src/runtime/mod.rs

```diff
@@ -211,27 +204,23 @@ pub struct Runtime {
     /// Task executor
     kind: Kind,
 
-    /// Handles to the network drivers
-    net_handles: Vec<io::Handle>,
-
-    /// Timer handles
-    timer_handles: Vec<timer::Handle>,
+    /// Handle to runtime, also contains driver handles
+    handle: Handle,
 
-    /// Blocking pool handle
-    #[cfg(feature = "blocking")]
-    blocking_pool: PoolWaiter,
+    /// Blocking pool handle, used to signal shutdown
+    blocking_pool: BlockingPool,
```

Comment:
The `BlockingPool` type imported in this file is actually from `tokio::runtime::blocking`, which is a shim supporting conditional compilation. Without the `blocking` feature, `BlockingPool` is an empty struct.